Encoding error latin-1 (Kiauh Alpha v6)

### Linux Distribution

Linux tronxy 6.6.51+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.51-1+rpt3 (2024-10-08) aarch64 GNU/Linux


### What happened

[ERROR] An unexpected error occured:
'latin-1' codec can't encode characters in position 0-56: ordinal not in range(256)
Traceback (most recent call last):
  File "/home/trash/kiauh/kiauh/core/menus/base_menu.py", line 184, in run
    self.display_menu()
  File "/home/trash/kiauh/kiauh/core/menus/base_menu.py", line 177, in display_menu
    print_header()
  File "/home/trash/kiauh/kiauh/core/menus/base_menu.py", line 50, in print_header
    print(header, end="")
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 0-56: ordinal not in range(256)




### What did you expect to happen

the new python menu appearing

### How to reproduce

start KIAUH

select yes on trying out v6

### Additional information

this was on a rpi5 everything is up to date on the system

locale info

LANG=en_US.UTF-8
LANGUAGE=en_US
LC_CTYPE="en_US"
LC_NUMERIC="en_US"
LC_TIME="en_US"
LC_COLLATE="en_US"
LC_MONETARY="en_US"
LC_MESSAGES="en_US"
LC_PAPER="en_US"
LC_NAME="en_US"
LC_ADDRESS="en_US"
LC_TELEPHONE="en_US"
LC_MEASUREMENT="en_US"
LC_IDENTIFICATION="en_US"
LC_ALL=en_US

--- a/kiauh.py
+++ b/kiauh.py
@@ -9,7 +9,13 @@
 #  This file may be distributed under the terms of the GNU GPLv3 license  #
 # ======================================================================= #
 
+import io
+import sys
+
 from kiauh.main import main
+
+# ensure that all output is utf-8 encoded
+sys.stdout = io.TextIOWrapper(sys.stdout.buffer, encoding="utf-8")
 
 if __name__ == "__main__":
     main()

